wormhole-connect/src/views/v2/Bridge/ReviewTransaction/index.tsx

```diff
@@ -53,6 +54,19 @@ const useStyles = makeStyles()((theme) => ({
   },
 }));
 
+const StyledButton = styled(Button)<ButtonProps>(({ theme }) => ({
```

Comment:
**Note**: I'll move this styled button into its own component as it's now shared among multiple locations.